File: src/main/java/com/github/wgzhao/dbquery/entities/DataSources.java
package com.github.wgzhao.dbquery.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name="data_sources")
@Setter
@Getter
public class DataSources extends BaseEntity
{

    @Id
    private String no;

    private String name;

    private String url;

    private String username;

    private String password;

    private String driver;

    @Override
    public String toString() {
        return String.format(" url: %s, \t" +
                " username: %s, \t" +
                " password: %s, \t",url, username, password);
    }
}

File: src/main/kotlin/com/github/wgzhao/dbquery/entities/DataSources.kt
package com.github.wgzhao.dbquery.entities

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "data_sources")
data class DataSources(
    @Id
    val no: String? = null,
    val name: String? = null,
    val url: String,
    val username: String? = null,
    val password: String? = null,
    val driver: String? = null
) : BaseEntity() {
    override fun toString(): String {
        return String.format(
            " url: %s, \t" +
                    " username: %s, \t" +
                    " password: %s, \t", url, username, password
        )
    }
}
